.changeset/tame-crabs-act.md

```diff
@@ -0,0 +1,13 @@
+---
+'@graphql-hive/gateway': minor
+---
+
+Improve `cache` configuration signature. 
+
+The `cache` configuration key now allow you to pass a custom factory function to get the cache instance: 
+
+```ts
+
+```
+
+In addition, the `context` exposed to cache functions now include the complete context of the gateway. 
```

Comment:
Context might be confusing here, because there is also GraphQL Context etc

Reply:
cleaned it a bit, thanks